app/src/androidTest/java/org/fossasia/susi/ai/login/LoginActivityTest.kt

```diff
@@ -74,11 +74,9 @@ class LoginActivityTest {
         onView(withId(R.id.customer_server)).check(matches(isDisplayed()))
 
         // checks if sign up button is present
-        onView(withId(R.id.sign_up)).perform(scrollTo())
```

Comment:
These should not be removed

Reply:
@iamareebjamal Actually, these statements were not there initially. I had added these because the issue with failing UI tests was "cannot find view with id..." So, as a beginner in testing I was trying to fix it that time. But, the tests are still failing. Should I add these back?